pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -811,8 +831,10 @@ struct LoweringPass : public mlir::ModulePass<LoweringPass> {
         EltwiseOpConversion<ew::BitOrOp, StdOp<mlir::OrOp>, OperandsAre<EltwiseInteger>>,
         EltwiseOpConversion<ew::BitXorOp, StdOp<mlir::XOrOp>, OperandsAre<EltwiseInteger>>,
         EltwiseOpConversion<ew::BitShlOp, StdOp<mlir::ShiftLeftOp>, OperandsAre<EltwiseInteger>>,
-        EltwiseOpConversion<ew::BitShrOp, StdOp<mlir::SignedShiftRightOp>, OperandsAre<EltwiseSigned>>,
-        EltwiseOpConversion<ew::BitShrOp, StdOp<mlir::UnsignedShiftRightOp>, OperandsAre<EltwiseUnsigned>>,
+        EltwiseOpConversion<ew::BitShrOp, StdOp<mlir::SignedShiftRightOp>,
+                            FirstOperandIsAndRestAre<EltwiseSigned, EltwiseInteger>>,
+        EltwiseOpConversion<ew::BitShrOp, StdOp<mlir::UnsignedShiftRightOp>,
+                            FirstOperandIsAndRestAre<EltwiseUnsigned, EltwiseInteger>>,
```

Comment:
Actually, it's only the first operand that we care about, the rest should always be unsigned; we should update the eltwise `ops.td` to reflect that. Once that's in, the predicate here would only be `FirstOperandIs<xxx>`